modules/monitoring-configurable-monitoring-components.adoc

```diff
@@ -2,53 +2,89 @@
 //
 // * observability/monitoring/configuring-the-monitoring-stack.adoc
 
-[id="configurable-monitoring-components_{context}"]
-= Configurable monitoring components
+:_mod-docs-content-type: REFERENCE
 
-This table shows the monitoring components you can configure and the keys used to specify the components in the
-ifndef::openshift-dedicated,openshift-rosa[]
-`cluster-monitoring-config` and
-endif::openshift-dedicated,openshift-rosa[]
-`user-workload-monitoring-config` `ConfigMap` objects.
```

Comment:
I forgot to divide the content in the previous PR, so I am doing it here. the main thing to check here is the rendering of the two procedures:

1) For core platform monitoring (CPM): link1, link2
2) For user workload monitoring (UWM): link